include/mbgl/ios/MGLMapView.h

```diff
@@ -364,6 +371,9 @@ IB_DESIGNABLE
 // TODO
 - (void)mapViewDidFinishRenderingMap:(MGLMapView *)mapView fullyRendered:(BOOL)fullyRendered;
 
+// TODO
+- (void)mapViewDidFinishLoadingAllAssets:(MGLMapView *)mapView;
+
```

Comment:
The other ones are mirrored on MapKit, and this one is close to these which exist (or will): 

```objc
- (void)mapViewWillStartLoadingMap:(MGLMapView *)mapView;
- (void)mapViewDidFinishLoadingMap:(MGLMapView *)mapView;
- (void)mapViewDidFailLoadingMap:(MGLMapView *)mapView withError:(NSError *)error;
- (void)mapViewWillStartRenderingMap:(MGLMapView *)mapView;
- (void)mapViewDidFinishRenderingMap:(MGLMapView *)mapView fullyRendered:(BOOL)fullyRendered;
```

"Assets" doesn't really have a context for regular map view of the map. If this is specific to benchmarking, we should make a `MGLBenchMarkDelegate` protocol which extends `MGLMapViewDelegate` (or, doesn't have to) for these purposes.